js/Events/commonEvents.js

```diff
@@ -22,7 +22,9 @@ interviewApp.Events.EventsCommon = function () {
         interviewApp.Events.EventsLogIn().remove();
     };
 
-    return { add: AddEventsCommon,
+    return {
+        add: AddEventsCommon,
         remove: RemoveEventsCommon,
-        removeAll: RemoveAllEvents };
+        removeAll: RemoveAllEvents,
```

Comment:
Any particular reason for the extra dangling comma?

Reply:
Autofix from ESLint does this. No ideea why.